pkg/storj/object.go

```diff
@@ -59,6 +59,8 @@ type Stream struct {
 	// FixedSegmentSize is the size of each segment,
 	// when all segments have the same size. It is -1 otherwise.
 	FixedSegmentSize int64
+	// SegmentList is segments info
+	SegmentList []Segment
```

Comment:
These aren't properties of the Stream, this is derived information and shouldn't be part of this.

Reply:
done